### ISSUE TITLE
feat: add plugin for maintaining mail accounts locally

### ISSUE THREAD

**Issue by ThorstenSuckow** [conjoon/conjoon#23]
Provide a user-plugin for using extjs-app-webmail with account data stored locally on the computer of the user, allowing for

 - adding/removing mail accounts
 - edit email account data

Adding mail account data should involve a process that lets the user select from a list of well known mail hosters.
Further functionality, such as subscribing to mailboxes, should be investigated once base implementation is done.

This ticket is related to conjoon/extjs-app-webmail#255 - the client should be able to switch between the plugins (e.g. extjs-app-imapuser) by adding the package-builds during runtime (and adjusting configurationf iles accordingly).

**Comment by ThorstenSuckow** [comment]
merge into php-lib-conjoon#8
